docs/docs/platform/cli.mdx

```diff
@@ -81,6 +81,18 @@ nhost up
 
 Hasura Console starts automatically and your Nhost app is running locally with the backend URL: `http://localhost:1337`.
 
+## Subdomain and Region
+
+Use `localhost:1337` as the `subdomain`, and skip `region` when using the CLI and the [JavaScript SDK](/reference/javascript):
```

Comment:
Probably best to reflect @nunopato's PR #786 and use `localhost` instead of `localhost:1337`

Reply:
Yes, but let's update the docs once that PR is merged.